app/util/PendingAction.scala

```diff
@@ -10,11 +10,11 @@ trait PendingAction[R] {
   /**
     * Completes the action.
     */
-  def complete(): Try[R]
+  def complete(implicit ec: ExecutionContext): Future[R]
```

Comment:
This is a side-effecting method, so reintroducing a the standard parenthesis would be nice.